app/controllers/tasks_controller.rb

```diff
@@ -1,4 +1,54 @@
 # frozen_string_literal: true
 
 class TasksController < ApplicationController
+  def index
+    @tasks = Task.all.sort_by(&:id)
+  end
+
+  def show
+    @task = Task.find(params[:id])
```

Comment:
In Rails controller, we would have a convention to put resource setter in `before_action` in order to reduce duplication. For example

```rb
before_action :set_foo, only: [:show, :edit]

....

private

def set_foo
  @foo = Foo.find(..)
end
```

Reply:
ref:
https://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-before_action
https://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-_insert_callbacks